src/subscript/field_statistics/field_statistics.py

```diff
@@ -314,6 +359,13 @@ def field_stat(args):
     if args.resultpath:
         relative_result_path = Path(args.resultpath)
     result_path = ens_path / relative_result_path
+    if not result_path.exists():
+        # Create the directory
+        logger.info(
+            "Create directory for results for field parameter statistics: "
+            f"{result_path}"
+        )
+        result_path.mkdir()
```

Comment:
We had discussion about this within subscript team regarding creating output path. I think majority of the code in subscript will trigger a warning that user should ensure the output path is already exist. There is discussion about removing the feature of automatic folder creating at some point.

https://github.com/equinor/subscript/blob/c6cef12c738dd3f80a7a2ca694c696c7081c9a03/src/subscript/prtvol2csv/prtvol2csv.py#L126

Reply:
I can of course replace it with an Error message if it does not exist, but it is easy to forget making the directory.
When running as a HOOK_WORKFLOW in ERT, a FORWARD model to create the directory will be, required. When running it from the workflow menu in ERT, the user have to first go to the scratch and make the directory manually.

I can also add a warning message that the directory is created if it is automatically created, and keep the automatic creating of the directory.

Reply:
It is considered better practice not to create directories implicitly, there are some downsides to it. Some code in subscript++ has had that feature previously, but it is typically deprecated or removed.

Reply:
OK, so I raise an Error then if not existing.